libs/labelbox/src/labelbox/schema/project_overview.py

```diff
@@ -0,0 +1,15 @@
+from typing import Dict, Optional
+from labelbox import pydantic_compat
+
+class Project_Overview(pydantic_compat.BaseModel):
```

Comment:
This should be CamelCase `ProjectOverview`

Reply:
Thanks. The name of the class was updated.